azure-pipeline-integration-test-steps-template.yaml

```diff
@@ -11,11 +11,6 @@ parameters:
   - ClassCTest
   
 steps:
-- task: UseDotNet@2
```

Comment:
Should we replace with install .net core 3?

Reply:
I needed to add this steps as .NET 2 is not present on all azure devops machines (same problem that we faced in the current project). .NET 3 is installed by default